aries-site/src/examples/components/pageheader/utils.js

```diff
@@ -5,6 +5,12 @@ export const groupActions = (actions, breakpoint, bestPractice) => {
   if (breakpoint === 'medium') {
     collapsedActions = actions.filter(action => !action.primary);
     displayedActions = actions.filter(action => action.primary);
+    // only leave primary and secondary visable
+  } else if (breakpoint === 'large') {
```

Comment:
The thought process for collapsedActions and DisplayedActions:

- Show all 3 buttons at xlarge
- Show overflow menu + 2 buttons at large
- Show overflow menu + primary button at medium

Reply:
This makes sense